File: ios/Plugin/CheckPlayServicesPlugin.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

// Define the plugin using the CAP_PLUGIN Macro, and
// each method the plugin supports using the CAP_PLUGIN_METHOD macro.
CAP_PLUGIN(CheckPlayServicesPlugin, "CheckPlayServices",
           CAP_PLUGIN_METHOD(getAvailability, CAPPluginReturnPromise);
           CAP_PLUGIN_METHOD(isAvailable, CAPPluginReturnPromise);
)

File: ios/Plugin/CheckPlayServicesPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(CheckPlayServicesPlugin)
public class CheckPlayServicesPlugin: CAPPlugin {
    private let implementation = CheckPlayServices()
    
    @objc func getAvailability(_ call: CAPPluginCall) {
        call.unavailable("Method nonsensical on iOS.")
    }
    
    @objc func isAvailable(_ call: CAPPluginCall) {
        call.unavailable("Method nonsensical on iOS.")
    }
}
